docs/topics/circuit-breaker-mission-intro-paragraph.adoc

```diff
@@ -1,3 +1,6 @@
-Mission proficiency level: xref:proficiency_foundational[*{proficiency-foundational}*].
+Mission proficiency level: 
+//special case since topic is used by front end.
+ifdef::docs-topic[xref:proficiency_foundational[*{proficiency-foundational}*].]
+ifndef::docs-topic[link:https://appdev.openshift.io/docs/wf-swarm-runtime.html#proficiency_levels[*Foundational*].]
```

Comment:
@edewit you shouldn't have to take any additional action. We built this so the topic itself could figure out if it was being used by a doc or the front end.